packages/documentation/pages/foundations/icons/typescript.vue

```diff
@@ -39,7 +39,7 @@ console.log(Yoco.Icon.ANNOUNCE)
 </template>
 
 <script lang="ts">
```

Comment:
This file seems weird with its double script tags

Reply:
one of them is an example block